shell/packages/sandstorm-ui-powerbox/powerbox-server.js

```diff
@@ -39,13 +46,13 @@ Meteor.methods({
         throw new Meteor.Error(403, "User must be an admin to powerbox offer IpNetwork");
       }
 
-      descriptor = "EAZQAQEAABEBF1EEAQH_QCAqemtXgqkAAAA";
+      descriptor = encodePowerboxDescriptor({tags: [{id: "12214421258504904768"}]});
```

Comment:
I honestly forgot we had .typeId constants. I've gone and replaced all the hard-coded IDs with these.